src/sharedUtils.ts

```diff
@@ -132,3 +132,14 @@ export const isInInactiveTake: (
 
 export const sleep: (seconds: number) => Promise<void> = (seconds) =>
   new Promise((resolve) => setTimeout(resolve, seconds * 1000));
+
+export const checkSentenceEnd = (word: Word): boolean => {
```

Comment:
Does this syntax work? I've seen it used for `function` declarations but surprised it works for const. If it works then all good but if it doesn't here's what we normally use

`export const checkSentenceEnd: (word: Word) => boolean = (word) => {`

Reply:
Idk, but I changed it to normal for consistency anyway